DataCompareCron/src/main/resources/application.yaml

```diff
@@ -0,0 +1,26 @@
+spring:
+  application:
+    name: DataCompareCron
+  main:
+    web-application-type: none  # Disables embedded web server
+  config:
+    activate:
+      on-profile: default
+
+data_exchange:
+  clientId: ${CLIENT_ID}
+  secret: ${CLIENT_SECRET}
+  url: ${DATA_EXCHANGE_URL}
+  endpoint_token: ${data_exchange.url}${DATA_EXCHANGE_TOKEN:/api/auth/token}
+  endpoint_compare: ${data_exchange.url}${DATA_EXCHANGE_COMPARE:/api/data-compare}
+
+scheduler:
+  cron: "0 */5 * * * *"
```

Comment:
give this as an Environment variable for user to decide
also you can set default value to Every one minute